fineract-loan/src/main/java/org/apache/fineract/portfolio/interestpauses/service/InterestPauseWritePlatformServiceImpl.java

```diff
@@ -178,9 +217,13 @@ private void validateVariations(Loan loan, LoanTermVariations variation) {
 
         List<LoanTermVariations> existingVariations = loan.getLoanTermVariations();
         for (LoanTermVariations existingVariation : existingVariations) {
-            if (!existingVariation.equals(variation) && variation.getTermApplicableFrom().isBefore(existingVariation.getDateValue())
-                    && variation.getDateValue().isAfter(existingVariation.getTermApplicableFrom())) {
-                throw new GeneralPlatformDomainRuleException("interest.pause.overlapping", "Overlapping interest pauses are not allowed.");
+            if (!existingVariation.equals(variation)
+                    && Objects.equals(existingVariation.getTermType().getValue(), INTEREST_PAUSE.getValue())) {
+                if (!variation.getTermApplicableFrom().isAfter(existingVariation.getDateValue())
```

Comment:
I am not sure about this conditions...
So both of the dates are inclusive, so if we want to create interest pause from 1 January to 2 January, the from date is 1 January and end date is 2 January.

If i create a new interest pause, i want to make sure the due date is **BEFORE** the existing variation from date or the from date is **AFTER** the existing variation due date, no?